pkg/config/config.go

```diff
@@ -170,21 +189,15 @@ func (gc *Config) addTrustedCerts(certChain string) (err error) {
 }
 
 func (gc *Config) configureProxies() {
-	if gc.httpProxy != "" {
```

Comment:
whats the thinking behind removing empty check? if it's to unset, shouldnt we be using Unsetenv?

Reply:
The thinking behind it is: We want to always just set the variable to whatever we are given. So creates, updates and deletes are all just a simple `os.Set()`

If they delete the config, this will be set to an empty string. 

We *could* check if its empty and do the `os.Unset()` but that felt like more code for the same outcome ("" and unset are equivalent)

Reply:
i dont think we have assurance from various tools that http_proxy= (empty) is equivalent to no http_proxy at all.

Reply:
hmm, fair enough. Perhaps that assumption was misinformed. It's probably safer to unset the variable if there is a doubt on the behaviour.